apps/server/src/modules/files/service/files.service.ts

```diff
@@ -22,6 +26,8 @@ export class FilesService {
 
 		await this.repo.save(entities);
 
+		this.logger.log({ action: 'Deleted Permissions To Any Files for user ', userId });
```

Comment:
Please log some more metadata (as for the _classes_ cases where I've left some example).

Reply:
OK I can only now add info about number of updated files without permission for deleted user. More deteiled information about updated documents will be added in ticket https://ticketsystem.dbildungscloud.de/browse/BC-6114